.gitignore

```diff
@@ -84,3 +84,9 @@ openapi/*
 
 # Cached License Data Files
 /.licenses
+
+# devenv.sh
+.devenv*
+
+# direnv.net
+.direnv
```

Comment:
Would it work for you if we only accept these changes? The worry I have here is that these tools are very specific to a particular workflow.

Reply:
Fair, I can live with my devenv.* file hanging around. Will submit another PR just for the .gitignore.